### ISSUE TITLE
The results cannot be replicated

### ISSUE THREAD

**Issue by Yintel12138** [megvii-research/mdistiller#62]
Title: Unable to Reproduce DKD Experiment Results on Tesla T4 Server Using Repository Code
Body:Dear maintainers,
I recently attempted to replicate the experiment results of Distilled Knowledge Distillation (DKD) mentioned in the paper, using your repository's code on my Tesla T4 server. Unfortunately, I was not able to achieve the same results as documented.
Could you please advise if there are any specific configurations or steps that I might have missed? Here is what I have done so far:
Set up the environment as per the documentation.
Pulled the latest code from the master branch of the repository.
Followed the instructions in the README file to set up the DKD experiment.
Ran the experiment with the default settings provided.
However, the results were significantly different from those reported in the paper. I would appreciate any guidance or recommendations to address this issue.
Thank you for your time and assistance.
Best regards,
Yintel
![image](https://github.com/megvii-research/mdistiller/assets/61579702/4fb3926c-e972-462f-aeb4-29957c3d586a)
![image](https://github.com/megvii-research/mdistiller/assets/61579702/58f1ed38-704d-42ce-838e-f87e38ec15e3)
![image](https://github.com/megvii-research/mdistiller/assets/61579702/4b37714c-0e85-40ab-800b-d8fb60517310)

**Comment by Zzzzz1** [comment]
Did you run the experiment on 8 GPUs? The batch-size on each GPU could be very small if running on 8 GPUs. The results reported for CIFAR-100 is run on only 1 GPU.

**Comment by Yintel12138** [comment]
Thank You. I will try it on only one gpu. Another question is, if I want to set a large batch size for boosting training speed, learning rate should increase or decrease?

**Comment by aaab8b** [comment]
> Thank You. I will try it on only one gpu. Another question is, if I want to set a large batch size for boosting training speed, learning rate should increase or decrease?

Increase the learning rate.
